app/javascript/react/components/MoviesIndexPage.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from "react"
+
+const MoviesIndexPage = (props) => {
+  const [movies, setMovies] = useState([])
+
+  useEffect(() => {
+    fetch("/api/v1/movies.json")
+    .then (response => {
+      if (response.ok) {
+        return response
+      } else {
+        let errorMessage = `${response.status} (${response.statusText})`,
+          error = new Error(errorMessage)
+        throw error
+      }
+    })
+    .then(response => response.json())
+    .then(body => {
+      setMovies(body) 
+    })
+    .catch(error => console.error(`Error in fetch: ${error.message}`))
+  }, [])
+
+  let movieListItems = movies.map((movie) => {
+    return <li>{movie.title} | {movie.summary} | {movie.year}</li>
+  })
+
+  return (
+    <div>
+      <ul>
+        {movieListItems}
+      </ul>
+    </div> 
+  )
+}
+
+export default MoviesIndexPage
```

Comment:
This whole file has nothing to do with the `show page` so it shouldnt be here

Reply:
repulled after other PR was approved, this shouldnt be a problem now i think